src/utils/string/StringUtils.js

```diff
@@ -188,7 +188,6 @@ const getChatName = (usernames, me) => {
   if (userArraysWithoutMe.length === 1) {
     return userArraysWithoutMe[0].trim();
   }
-  return 'No name';
 };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The function `getChatName` no longer returns 'No name' when `userArraysWithoutMe.length !== 1`. This could potentially introduce bugs if other parts of the code expect 'No name' to be returned in this case. Make sure this change is reflected throughout your codebase.



<!-- This is an auto-generated comment by OSS CodeRabbit -->